Assembler/js65/macpack/common.js

```diff
@@ -1,4 +1,75 @@
 export const text = `
+;;; Standard register names
```

Comment:
Add standard register names so the codebase can get rid of explicit register numbers.